webui/react/src/App.tsx

```diff
@@ -171,16 +133,18 @@ const AppView: React.FC = () => {
             <>
               {isServerReachable ? (
                 <ConfirmationProvider>
+                  {/* Global app components: */}
```

Comment:
what's this change for?

Reply:
no functional reason, and not strictly necessary, just seemed to make sense for all of these to be siblings of `Navigation`